diagnostic.md

```diff
@@ -157,6 +185,16 @@ div {
 ```
 
 <!-- your answer starts here -->
+All of the divs except for divFour will be in a line on the top.
+divOne will be all the way on the right because it floats right.
+The other divs will be lined up from left to right in order, with divOne on the right.
+
+divFour will float to the left, but will be in its own row under the others (all the way to the left).
+
+I think the arrangement looks like this:
+
+divTwo, divThree, divFive, divOne
+divFour
```

Comment:
Correct order is:

2,3,1
4,5

Since div 2,3,1 are all floated, they will be on the same line. 
div4 is cleared so it's moved to a new line, but has a float from the div class and div5 is floated; thus them being on the same row.